sqlflow_models/dnnclassifier.py

```diff
@@ -0,0 +1,42 @@
+import tensorflow as tf
+
+class DNNClassifier(tf.keras.Model):
+    def __init__(self, feature_columns, hidden_units=[10,10], n_classes=2):
+        """DNNClassifier
+        :param feature_columns: feature columns.
+        :type feature_columns: list[tf.feature_column].
+        :param hidden_units: number of hidden units.
+        :type hidden_units: list[int].
+        :param n_classes: List of hidden units per layer.
+        :type n_classes: int.
+        """
+        super(DNNClassifier, self).__init__()
+
+        # combines all the data as a dense tensor
+        self.feature_layer = tf.keras.layers.DenseFeatures(feature_columns)
+        self.hidden_layers = []
+        for hidden_unit in hidden_units:
+            self.hidden_layers.append(tf.keras.layers.Dense(hidden_unit))
+        self.prediction_layer = tf.keras.layers.Dense(n_classes, activation='softmax')
+
+    def call(self, inputs):
+        x = self.feature_layer(inputs)
+        for hidden_layer in self.hidden_layers:
+            x = hidden_layer(x)
+        return self.prediction_layer(x)
+
+    def default_optimizer(self):
+        """Default optimizer name. Used in model.compile."""
+        return tf.keras.optimizers.Adagrad(lr=0.1)
+
+    def default_loss(self):
+        """Default loss function. Used in model.compile."""
+        return 'sparse_categorical_crossentropy'
+
+    def default_training_epochs(self):
+        """Default training epochs. Used in model.fit."""
+        return 2
```

Comment:
`2` might be too small. Change to `50`?

Reply:
Tested 2 is enough for iris demo case. Change to 50 maybe take too long to train